libnavui-dropin/src/test/java/com/mapbox/navigation/dropin/navigationview/MapboxNavigationViewApiTest.kt

```diff
@@ -334,6 +350,16 @@ class MapboxNavigationViewApiTest {
         verify { testStore.dispatch(TripSessionStarterAction.EnableTripSession) }
     }
 
+    @Test
+    fun `getCurrentVoiceInstructionsPlayer returns valid player instance`() {
+        every { audioGuidance.getCurrentVoiceInstructionsPlayer() } returns null
+        assertNull(sut.getCurrentVoiceInstructionsPlayer())
+
+        val player: MapboxVoiceInstructionsPlayer = mockk()
+        every { audioGuidance.getCurrentVoiceInstructionsPlayer() } returns player
```

Comment:
I'd also check that the returned instance changes when `audioGuidance.getCurrentVoiceInstructionsPlayer()` changes from non-null to another non-null value. Just to make sure it's not a lazy creation approach.